src/sentry/search/django/backend.py

```diff
@@ -169,6 +171,20 @@ def _build_queryset(self, project, query=None, status=None, tags=None,
                     params['last_seen__lt'] = last_seen_to
             queryset = queryset.filter(**params)
 
+        if active_at_from or active_at_to:
```

Comment:
these likely aren't indexed well, so we might want to move this into a subquery to limit the depth of the search